src/@chakra-ui/gatsby-plugin/semanticTokens.ts

```diff
@@ -34,12 +34,17 @@ const semanticTokens = {
     primaryLight: { _light: "blue.100", _dark: "orange.100" },
     primaryDark: { _light: "blue.700", _dark: "orange.800" },
     primaryHover: { _light: "blue.300", _dark: "orange.300" },
+    primaryPressed: { _light: "blue.300", _dark: "orange.800" },
     body: { _light: "gray.700", _dark: "gray.100" },
+    bodyInverted: { _light: "gray.100", _dark: "gray.700" },
     bodyLight: { _light: "gray.500", _dark: "gray.100" },
     disabled: { _light: "gray.400", _dark: "gray.500" },
     background: { _light: "white", _dark: "gray.700" },
-    success: "green.500",
+    neutral: { _light: "white", _dark: "gray.900" },
+    success: { _light: "green.500", _dark: "green.400" },
+    successNeutral: { _light: "green.100", _dark: "green.900" },
     error: "red.500",
+    errorNeutral: { _light: "red.100", _dark: "red.900" },
```

Comment:
cc: @nloureiro @konopkja
Take a look and let me know if you disagree with anything here, or if you have suggestions on any naming improvements. If we keep these we may want to update the DS with the ones that are missing.